packages/create-toolpad-app/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "create-toolpad-app",
+  "version": "0.0.42-alpha.5",
+  "keywords": [
+    "react",
+    "toolpad",
+    "mui"
+  ],
+  "description": "Create Toolpad apps with one command",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mui/mui-toolpad",
+    "directory": "packages/create-toolpad-app"
+  },
+  "author": "Toolpad Team",
+  "license": "MIT",
+  "bin": {
+    "create-toolpad-app-test": "./dist/index.js"
+  },
+  "files": [
+    "dist"
+  ],
+  "type": "module",
```

Comment:
```suggestion
```
?

Reply:
Without this I get 
```
SyntaxError: Cannot use import statement outside a module
``` 
errors when running the compiled `.js` file

Reply:
Fixed this by dynamically importing all the ES modules (`chalk`, `inquirer` and `execa`)